src/components/BingoBoardCard.vue

```diff
@@ -90,27 +113,33 @@ function reset() {
     }
   }
   clearInterval(startBingo.value);
-  show.value = false;
+  showBingo.value = false;
   bingoNumbers.value = newNumbers;
+  includedColumnIndexes.value = [];
 }
 reset();
+
+function getColumnStyle(index) {
+  return {
+    backgroundColor: includedColumnIndexes.value.includes(index)
+      ? "green"
+      : "transparent",
+    color: includedColumnIndexes.value.includes(index) ? "white" : "black",
+  };
+}
 </script>
 
 <style scoped>
 .box {
-  /* background-color: lightgrey; */
-  color: #000000;
-  border-radius: 5px;
+  color: black;

```

Comment:
create color variables and use color variables

Reply:
okay